src/mango/src/mango_cursor_view.erl

```diff
@@ -56,18 +56,31 @@ create(Db, Indexes, Selector, Opts) ->
 
 explain(Cursor) ->
     #cursor{
-        index = Idx,
-        ranges = Ranges
+        opts = Opts
     } = Cursor,
-    case Ranges of
-        [empty] ->
-            [{range, empty}];
-        _ ->
-            [{range, {[
-            {start_key, mango_idx:start_key(Idx, Ranges)},
-            {end_key, mango_idx:end_key(Idx, Ranges)}
-        ]}}]
-    end.
+
```

Comment:
Previous implementation returned `range` property as well. We don't return it anymore. Would it be a problem for fauxton?

Reply:
fauxton just displays the raw JSON result from `_explain`, so it won't be a problem. The `range` property is now part of the `mrargs` object in the result.